CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 2.38.0-rc.0 / 2022-08-11
+
+* [FEATURE]: Web: Add a `/api/v1/format_query` HTTP API endpoint that allows pretty-formatting PromQL expressions. #11036 #10544 #11005
+* [FEATURE]: UI: Add support for formatting PromQL expressions in the UI. #11039
+* [FEATURE]: DNS SD: Support MX records for discovering targets. #10099
+* [FEATURE]: Templates: Add `toTime()` template function that allows converting sample timestamps to Go `time.Time` values. #10993
+* [ENHANCEMENT]: Kubernetes SD: Add `__meta_kubernetes_service_port_number` meta label indicating the service port number. #11002
```

Comment:
Thanks, added. I also merged the other docs change and included it here. I fast-forwarded the release branch to `main`, since nothing has been cut yet and there were only two docs changes.